packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -1274,8 +1285,7 @@ class PubspecDependency extends PubspecLine {
   /// dependencies/dev_dependencies section, or a dependency_overrides section.
   /// We track this so that we can put ourselves in the right section when
   /// generating the fake pubspec.yaml.
-  bool get lockIsOverride => _lockIsOverride;
-  late bool _lockIsOverride;
+  bool? _lockIsOverride;
```

Comment:
I deleted the public getter, since:

1. all the logic for this is in a single library, so it didn't matter that it was public
2. it obscured my editor reference list, since assignments were to the private field, but accesses were to the public getter.